terraform/image-proxy/main.tf

```
@@ -0,0 +1,20 @@
+variable "domain" { type = string }
+variable "sitename" { type = string }
+variable "conveyor_address" { type = string }
+
+
+resource "fastly_service_vcl" "test-pypi-camo" {
```

Since this is a module, we don't need to specify "test" here.

```suggestion
resource "fastly_service_vcl" "camo" {
```